nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/SetPropertiesAction.java

```diff
@@ -45,12 +48,16 @@
 
     public static final String ACTION_NAME = "setProperties";
 
+    // duplicated from NXAuditEventsService.DISABLE_AUDIT_LOGGER
+    public static final String PARAM_DISABLE_AUDIT = "disableAuditLogger";
+
+    public static final String PARAM_VERSIONING_OPTION = VersioningService.VERSIONING_OPTION;
+
     @Override
     public Topology getTopology(Map<String, String> options) {
         return Topology.builder()
-                       .addComputation(SetPropertyComputation::new,
-                               Arrays.asList(INPUT_1 + ":" + ACTION_NAME, //
-                                       OUTPUT_1 + ":" + STATUS_STREAM))
+                       .addComputation(SetPropertyComputation::new, //
+                               Arrays.asList(INPUT_1 + ":" + ACTION_NAME, OUTPUT_1 + ":" + STATUS_STREAM))
```

Comment:
you break an intentional formatting that is trying to get readable mapping

Reply:
My IDE broke it, I miss-fixed it, rolling back